src/components/vm/nics/nicBody.jsx

```diff
@@ -203,6 +203,31 @@ export const NetworkTypeAndSourceRow = ({ idPrefix, onValueChanged, dialogValues
                     </FormSelect>
                 </FormGroup>
             )}
+            {dialogValues.networkType == "direct" && (
+                <FormGroup id={`${idPrefix}-source-mode`} label={_("Mode")} hasNoPaddingTop isInline
+                           data-value={dialogValues.networkSourceMode}>
+                    <Radio id={`${idPrefix}-source-mode-vepa`}
+                        name="mode-vepa"
+                        isChecked={dialogValues.networkSourceMode == "vepa"}
+                        label={_("VEPA")}
```

Comment:
I don't think we can translate acronyms right?

Reply:
Fixed.